src/main/java/core/basesyntax/service/RegistrationServiceImpl.java

```diff
@@ -9,6 +10,27 @@
 
     @Override
     public User register(User user) {
-        return null;
+        if (user == null) {
+            throw new RegistrationValidationException("User cannot be null");
+        }
+        if (user.getLogin().length() < 6) {
+            throw new RegistrationValidationException("Login must contain at least 6 characters");
+        }
+        if (user.getPassword().length() < 6) {
+            throw new
+                    RegistrationValidationException("Password must contain at least 6 characters");
+        }
+        if (user.getAge() < 18) {
```

Comment:
The age check should include a condition to handle negative values for age, as they are not valid and should result in an exception.